scripts/feature_tests/README.md

```diff
@@ -136,35 +136,35 @@ Recommended resource requirements for the host machine are: 8C CPUs, 32 GB RAM,
 
 We are running two main jobs for the **remediation/pivoting** framework testing.
```

Comment:
Would be nice also if you could update the test name mentions we run on feature tests in this whole document where we mention them, looks like they are missing some of them. We run:
- inspection ==> remediation ==> pivoting ==> node reuse ==> repivoting 

in feature tests